model_tuning/performance.py

```diff
@@ -4,13 +4,24 @@
 import time
 from typing import List
 
+import hnswlib
 from sentence_transformers import SentenceTransformer
-from sentence_transformers import util
-from torch import Tensor
 
+# MODEL VARIABLES
 MODEL_NAME = "intfloat/e5-base-v2"
+EMBEDDING_SIZE = 768
+
+# EMBEDDING VARIABLES
 EMBEDDING_CACHE_DIR = "../data/training_files/embeddings/"
 EMBEDDING_FILE = "loinc_lab_names_intfloat_e5-base-v2_20251007"
+
+# ANN INDEX VARIABLES
+INDEX_FP = "./hnswlib.index"
+EF_CONSTRUCTION = 200
+M_VALUE = 64
+EF_SEARCH = 100
```

Comment:
Can these variables be consistently names between hnsw_estimator.py and this script?